test/asm/break.err

```diff
@@ -2,4 +2,5 @@ warning: break.asm(9): [-Wuser]
     done 5
 warning: break.asm(17): [-Wuser]
     OK
-error: Unterminated IF construct (1 levels)!
+FATAL: break.asm(18) -> break.asm::REPT~1(23):
+    Ended block with 1 unterminated IF constructs
```

Comment:
Do we want to bother with that `s`?

Reply:
Just do `construct(s)` if it's too hard to handle proeprly

Reply:
`constructs` is wrong for 1; `construct` is wrong for 2, 3, etc. `construct(s)`?

Reply:
```c
		fatalerror("Ended block with %" PRIu32 " unterminated IF construct%s\n",
			   contextStack->nIFDepth, contextStack->nIFDepth == 1 ? "" : "s");
```